### PR TITLE
doc/designs/id-mapping.md: expand on ID range allocation details

### PR DESCRIPTION
**Pull request by abbra**
Related: https://pagure.io/freeipa/issue/9477

### CODE REVIEW

File: doc/designs/id-mapping.md [hunk 1]
```diff
@@ -158,6 +158,74 @@ In case of explicit allocation, ID range associated with the trusted forest
...
+an Active Directory, there would be two possibilities, managed within SSSD and
+IPA:
+
+ - an ID range is implicit, defined through a murmurhash3 hash of the domain
+   SID,
+
+ - an ID range is explicit, defined by the admin as UID/GID values stored in AD
+   LDAP user entries for each user
+
+The explicitly defined ID ranges most likely are part of a legacy setup and
```
Comment (rcritten) [inline doc/designs/id-mapping.md:197]: Where are these defined? As in a user-create idrange?

**Comment by rcritten** [comment]
Thanks for the patch, this overall looks good. Just one question. The explicit range sentence seems to lack context.

File: doc/designs/id-mapping.md (hunk 1 above)
Comment (abbra) [reply doc/designs/id-mapping.md:197]: No, as I said in a sentence just above this one -- "defined by the admin as UID/GID values stored in AD LDAP user entries for each user". They are explicit by the fact that each relevant user or group entry has UID and GID values in the LDAP attributes in AD LDAP tree.

**Comment by flo-renaud** [comment]
LGTM. Since the doc was added to ipa-4-10 and ipa-4-11 branches (PR #7073) I am adding the same labels

File: doc/designs/id-mapping.md (hunk 1 above)
Comment (rcritten) [reply doc/designs/id-mapping.md:197]: Ok, I guess I'd prefer it to be part of the bullet point since it only applies to that and not to AD in general.

**Comment by rcritten** [comment]
master:

* d4ffc53b2a3534d4f6c12e150fdfb3cfcb11cbae doc/designs/id-mapping.md: expand on ID range allocation details

**Pull request closed without merging**